Base/etc/FileIconProvider.ini

```
@@ -33,3 +33,4 @@ text=*.txt
 truetype=*.ttf
 pixelpaint=*.pp
 archive=*.zip,*.gz,*.tar
+video=*.webm
```

```suggestion
video=*.avi,*.mkv,*.mov,*.mp4,*.webm
```
While we're there :^)